deploy/bootstrap/defs.sh

```diff
@@ -51,8 +51,14 @@ function rm_politely {
 }
 
 function generate_password {
+    local force=${2:-no}
+    if [[ "${force}" != "force" ]] && [[ "${DEPLOY_DEV}" = "yes" ]]; then
+	echo "dummy"
+	return 0 # early return
+    fi
+    # Otherwise
     local size=${1:-16} # defaults to 16 characters
-    ${PYTHONEXEC} -c "import secrets,string;print(''.join(secrets.choice(string.ascii_letters + string.digits) for i in range(${size})))"
+    ${PYTHONEXEC:-python3.6} -c "import secrets,string;print(''.join(secrets.choice(string.ascii_letters + string.digits) for i in range(${size})))"
```

Comment:
maybe we should do a simple check for the python version in this script?
I use `3.7` as my default now

Reply:
Yes. Sure.

I have put 3.6 because it used to run inside the container, which is still on 3.6.
I'm also on 3.7 now.

However, the default is in the [run.sh script](https://github.com/EGA-archive/LocalEGA/blob/feature/tls-client-verification/deploy/bootstrap/run.sh#L22).